CMakeLists.txt

```diff
@@ -103,21 +101,33 @@ if (${IAS_IS_HOST_BUILD})
   include(CMakeLists.avb_tests.txt)
 endif()
 
+#------------------------------------------------------------------
+#Install
+#------------------------------------------------------------------
+
+install(CODE "execute_process(COMMAND cp ${CMAKE_CURRENT_SOURCE_DIR}/deps/gptp/linux/build/obj/daemon_cl ${CMAKE_INSTALL_PREFIX}/bin/daemon_cl)")
+install(CODE "execute_process(COMMAND mkdir -p /etc/alsa/conf.d)")
+
+install(CODE "execute_process(COMMAND cp ${CMAKE_CURRENT_SOURCE_DIR}/public/res/51-smartx.conf ${CMAKE_CURRENT_SOURCE_DIR}/public/res/51-smartx.conf.tmp)")
+install(CODE "execute_process(COMMAND sed -i s@<lib_path>@${CMAKE_INSTALL_LIBDIR}@g ${CMAKE_CURRENT_SOURCE_DIR}/public/res/51-smartx.conf.tmp)")
```

Comment:
Maybe use cmake configure_file (https://cmake.org/cmake/help/v3.0/command/configure_file.html) in place of this sed?